requirements.txt

```diff
@@ -1,5 +1,5 @@
 Jinja2==2.11.1
-lxml==4.5.0
+lxml==4.5.2
```

Comment:
We should not rely on fixed version unless necessary ; particularly for this which is shared with scraperlib.